.travis.yml

```diff
@@ -21,12 +21,13 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
+        - ASTROPY_USE_SYSTEM_PYTEST=1
         - PYTHON_VERSION=3.5
         - NUMPY_VERSION=stable
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='pytest>=3'
```

Comment:
Actually I added this out of desperation. Without this, most (all?) the Python 3 test suits fall back on pytest 2.7 (even older than the bundled version). It is unpredictable (at least not to me).